admin-dev/themes/new-theme/js/components/form/submittable-input.js

```diff
@@ -176,6 +211,10 @@ export default class SubmittableInput {
     const initialValue = $(input).data('initial-value');
     let newValue = $(input).val();
 
+    if ($(input).hasClass('is-invalid')) {
+      $(input).removeClass('is-invalid');
+    }
```

Comment:
```suggestion
    $(input).removeClass('is-invalid');
```

You can try to remove it even if there is no class on it :D